core/src/main/java/org/mule/util/xmlsecurity/DefaultXMLSecureFactories.java

```diff
@@ -19,6 +20,7 @@
 import javax.xml.transform.TransformerFactory;
 import javax.xml.validation.SchemaFactory;
 import javax.xml.validation.Validator;
+import javax.xml.XMLConstants;
```

Comment:
Unused import

Reply:
Not really. This class is being used in javadocs.